src/trade/bid/bid.resolver.ts

```diff
@@ -8,7 +13,15 @@ import { Bid } from './entities/bid.entity'
 
 @Resolver(() => Bid)
 export class BidResolver {
-    constructor(private readonly bidsService: BidService) {}
+    constructor(
+        private readonly bidsService: BidService,
+        private readonly emitter: EventEmitter2,
+        private readonly notifi: NotificationService,
```

Comment:
resolver `Bid` must know nothing about Notification module, use EventEmitter for create notification.